Java/E-montazysta/src/main/java/com/emontazysta/model/ToolType.java

```diff
@@ -26,6 +25,8 @@ public class ToolType {
     private String name;
 
     private int criticalNumber;
+    private int inServiceCount;
+    private int availableCount;
```

Comment:
```suggestion
```
These are a derived attributes, that are calculated in mapper.
https://github.com/s20146/PRO/blob/23a8b9a0ff4f1cd696ca79848103ed664bd4da22/Java/E-montazysta/src/main/java/com/emontazysta/mapper/ToolTypeMapper.java#L28